minecraft_servers/voasis/manifest.json

```diff
@@ -0,0 +1,61 @@
+{
+  "server_name": "voasis",
+  "nice_name": "Voasis",
+  "direct_ip": "voasis.de",
+  "server_wildcards": [
+    "%.laby.net"
+  ],
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "",
+    "web_shop": "",
+    "web_support": "",
+    "twitter": "",
+    "tiktok": "",
+    "facebook": "",
+    "instagram": "",
+    "discord": "https://discord.gg/Ux8zRjhNyZ",
+    "youtube": "",
+    "teamspeak": ""
+  },
+  "gamemodes": {
+    "TheBridge": {
+      "name": "TheBridge Alt", 
+      "color": "#FFA500",
+      "url": "",
+      "command": "/warteschlange beitreten TheBridge",
+      "versions": "1.20.6"
+    },
+    "TheBridgeFast": {
+      "name": "TheBridge Neu",
+      "color": "#ADD8E6",
+      "url": "",
+      "command": "/warteschlange beitreten TheBridge-Fast",
+      "versions": "1.20.6"
+    }
+  },
+  "chat": {
+    "message_formats": [
+      "<prefix><player><message>"
```

Comment:
Not a real regex, this is an **example** regex for the message format
`^§[a-f0-9](?<level>\\d+)(| §[a-f0-9](?<rank>\\S+)) §e(?<sender>[a-zA-Z0-9_]{2,16}) §f(?<message>.*)$`